packages/zipkin/index.d.ts

```diff
@@ -120,7 +116,7 @@ declare namespace zipkin {
   }
 
   namespace model {
-    interface Endpoint {
+    class Endpoint {
```

Comment:
note why class not interface? (this is me curious not saying it should be interface)

Reply:
If it’s an interface, then you can’t say `new Endpoint`, and the `constructor` is incorrectly interpreted as a method returning `any`.